Dockerfile

```diff
@@ -10,17 +10,22 @@ COPY go.sum go.sum
 # and so that source changes don't invalidate our downloaded layer
 # RUN go mod download
 
+COPY hack/ hack/
+COPY Makefile.specialresource.mk Makefile.specialresource.mk
+COPY Makefile.helm.mk Makefile.helm.mk
+COPY Makefile.helper.mk Makefile.helper.mk
+COPY Makefile Makefile
+
```

Comment:
We are missing:
```
RUN ["apt", "update"]
RUN ["apt", "install", "-y", "patch"]
```

This come from the CI:
```
bash: patch: command not found
```

Reply:
Forgot, it was on the other (now removed) Dockerfile.